tests/e2e-test.spec.js

```diff
@@ -18,15 +18,17 @@ const waitForDbt = async (pgClient, retry = 50) => {
     const dbtPersons = await pgClient.query(`SELECT * FROM ${DBT_POSTGRES_SCHEMA}.person`);
     const expectedPersons = persons().length * dbNames.length;
 
+    console.log(`Data records: ${dbtDataRecords.rows.length}/${expectedDataRecords}`);
+    console.log(`Persons: ${dbtPersons.rows.length}/${expectedPersons}`);
     if (dbtDataRecords.rows.length === expectedDataRecords && dbtPersons.rows.length === expectedPersons) {
       return;
     }
-  } catch {
-    // not done yet
+  } catch (error) {
+    console.log(error);
```

Comment:
From what I understand, this is not an error; it means DBT has yet to finish.

Reply:
Yes that is correct. I still found it helpful to see the logs while running the tests.